### DISCUSSION TITLE
Static Heatmap support

### DISCUSSION THREAD

**Discussion by paulorb** [visgl/deck.gl#7191] [Q&A]
I am trying to configure the heatmap as static (so it doesn't dynamically update as the user zooms in and out) similarly to what can be achieved on the ArcGis maps (see link below). Do you know if deck.gl supports this kind of configuration?

Links:
[ArcGis Static Heatmap](https://developers.arcgis.com/javascript/latest/sample-code/visualization-heatmap-static/)

Thanks,
Paulo

**Comment by Pessimistress** [comment]
Feel free to create a new feature request in issues.
